src/docs/development/ui/animations/implicit-animations.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Implicit Animations
+description: Where to find more information on using implicit animations in Flutter.
+---
+
+With Flutter's [animation library],
+you can add motion and create visual effects
+for the widgets in your UI.
+One part of the library is an assortment of widgets
+that manage animations for you.
+These widgets are collectively referred to as _implicit animations_,
+or _implicitly animated widgets_, deriving their name from the
+[ImplicitlyAnimatedWidget] class that they implement.
+The following set of resources provide many ways to learn
+about implicit animations in Flutter.
+
+## Documentation
+- [Implicit animations codelab] - Jump right into the code!
```

Comment:
Could you put a definition list here?

Global comment. Could you code font class names? That's what the style guide recommends and I'm trying to move towards that.  https://developers.google.com/style/code-in-text

Reply:
> Could you put a definition list here?

Of course! Do you have any recommendations on what we should define? My understanding is that this is a landing page for browsing content related to implicit animations. Given the preceding paragraph, what might be helpful to define on this landing page?

Reply:
> Could you code font class names? 

Absolutely -- do you see any classnames that aren't already being used as a link? I can't find any I've missed.

Reply:
No, you misunderstand. I'm talking strictly about formatting.  Try this:

[Implicit animations codelab]
: Jump right into the code! This codelab uses interactive examples 
   and step-by-step instructions to teach you how to use implicit animations.

[AnimatedContainer sample]
: A step-by-step recipe from the [Flutter cookbook]
  for using the [`AnimatedContainer`] implicitly animated widget.

And so on. (Note that I also tightened the prose.)

Reply:
Any time you see a list like this:

* stuff - define stuff

Then you really shouldn't use a bulleted list. You should use a definition list.

Reply:
Done! Thanks for the clarification.